test/ZoraNFTCreatorV1_Fork.t.sol

```diff
@@ -9,62 +10,157 @@ import {MockMetadataRenderer} from "./metadata/MockMetadataRenderer.sol";
         }
+
+        assertEq(
+            deployment.dropMetadata,
+            address(creator.dropMetadataRenderer()),
+            string.concat("configured drop metadata renderer incorrect on: ", chainName)
+        );
+        assertEq(
+            deployment.editionMetadata,
+            address(creator.editionMetadataRenderer()),
+            string.concat("configured edition metadata renderer incorrect on: ", chainName)
+        );
+        assertEq(deployment.dropImplementation, address(creator.implementation()), string.concat("configured metadata renderer incorrect on: ", chainName));
+    }
+
+    function forkEdition() internal {
+        address deployedEdition = creator.createEdition(
+            "name",
+            "symbol",
+            100,
+            500,
+            DEFAULT_FUNDS_RECIPIENT_ADDRESS,
+            DEFAULT_FUNDS_RECIPIENT_ADDRESS,
+            IERC721Drop.SalesConfiguration({
+                publicSaleStart: 0,
+                publicSaleEnd: type(uint64).max,
+                presaleStart: 0,
+                presaleEnd: 0,
+                publicSalePrice: 0.1 ether,
+                maxSalePurchasePerAddress: 0,
+                presaleMerkleRoot: bytes32(0)
+            }),
+            "desc",
+            "animation",
+            "image"
+        );
+        ERC721Drop drop = ERC721Drop(payable(deployedEdition));
+        (, uint256 fee) = drop.zoraFeeForAmount(10);
+        vm.startPrank(DEFAULT_FUNDS_RECIPIENT_ADDRESS);
+        vm.deal(DEFAULT_FUNDS_RECIPIENT_ADDRESS, 10 ether + fee);
+        drop.purchase{value: 1 ether + fee}(10);
+        assertEq(drop.totalSupply(), 10);
+    }
+
+    function forkDrop() internal {
+        address deployedDrop = creator.createDrop(
+            "name",
+            "symbol",
+            DEFAULT_FUNDS_RECIPIENT_ADDRESS,
+            1000,
+            100,
+            DEFAULT_FUNDS_RECIPIENT_ADDRESS,
+            IERC721Drop.SalesConfiguration({
+                publicSaleStart: 0,
+                publicSaleEnd: type(uint64).max,
+                presaleStart: 0,
+                presaleEnd: 0,
+                publicSalePrice: 0,
+                maxSalePurchasePerAddress: 0,
+                presaleMerkleRoot: bytes32(0)
+            }),
+            "metadata_uri",
+            "metadata_contract_uri"
+        );
+        ERC721Drop drop = ERC721Drop(payable(deployedDrop));
+        (, uint256 fee) = drop.zoraFeeForAmount(10);
+        drop.purchase{value: fee}(10);
+        assertEq(drop.totalSupply(), 10);
+    }
+
+    function forkDropGeneric() internal {
+        MockMetadataRenderer mockRenderer = new MockMetadataRenderer();
+        address deployedDrop = creator.setupDropsContract(
+            "name",
+            "symbol",
+            DEFAULT_FUNDS_RECIPIENT_ADDRESS,
+            1000,
+            100,
+            DEFAULT_FUNDS_RECIPIENT_ADDRESS,
+            IERC721Drop.SalesConfiguration({
```

Comment:
minor but i wonder if this can be a share value so we don't copy and paste it between the three tests, like have a "defaultSalesConfiguration"